webrtc.html

```diff
@@ -7484,14 +7485,15 @@ <h2>
         possible designs have been discussed in <a href=
         "https://github.com/w3c/webrtc-pc/issues/1283">GitHub issue 1283</a>.
       </p>
-      <p>
+      <p id="rtp-video-rescaled">
         When video is rescaled, for example for certain combinations of width
         or height and {{RTCRtpEncodingParameters/ scaleResolutionDownBy}}
         values, situations when the resulting width or height is not an integer
```

Comment:
Hmm, should we limit this sentence to `scaleResolutionDownBy`, to not step on RFC8829?

This seems to match better the extent of this spec's concerns, which I think are that `scaleResolutionDownBy` behaves deterministically.